src/process_stash.cpp

```diff
@@ -0,0 +1,84 @@
+#include "process_stash.hpp"
+
+#include <boost/uuid/uuid.hpp>
+#include <boost/uuid/uuid_generators.hpp>
+#include <boost/uuid/uuid_io.hpp>
+
+#include <iterator>
+#include <optional>
+#include <shared_mutex>
+#include <unordered_map>
+
+namespace
+{
+    // A mapping containing handles to heterogenous objects.
```

Comment:
heterogenous -> heterogeneous

Reply:
no worries - can tweak it now or later.

Reply:
Later!

Reply:
oh, it was already in the irods/irods version... ignore me!